go-controller/pkg/ovn/egressfirewall_test.go

```diff
@@ -455,47 +819,47 @@ var _ = Describe("OVN test basic functions", func() {
 				ipv6source:   "",
 				destinations: []matchTarget{{matchKindV4CIDR, "1.2.3.4/32"}},
 				ports:        nil,
-				output:       `match="(ip4.dst == 1.2.3.4/32) && ip4.src == $testv4 && ip4.dst != 10.128.0.0/14"`,
+				output:       `match="(ip4.dst == 1.2.3.4/32) && ip4.src == $testv4 && inport == \"jtor-ovn_cluster_router\""`,
```

Comment:
could `jtor-ovn_cluster_router` be built with the types in const.go? `\"" + types.JoinSwitchToGWRouterPrefix + types.OVNClusterRouter + "\"" `

Reply:
fixed. Thanks for your comments.